src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -645,8 +646,10 @@ struct GhValenciaDivCleanTemplateBase<
         tmpl::pair<TimeSequence<std::uint64_t>,
                    TimeSequences::all_time_sequences<std::uint64_t>>,
         tmpl::pair<TimeStepper, TimeSteppers::time_steppers>,
-        tmpl::pair<Trigger, tmpl::append<Triggers::logical_triggers,
-                                         Triggers::time_triggers>>>;
+        tmpl::pair<
+            Trigger,
+            tmpl::append<Triggers::logical_triggers, Triggers::time_triggers,
+                         tmpl::list<Triggers::SeparationLessThan<true>>>>>;
```

Comment:
This will add the trigger for all GhValencia execs. We probably only want this for the BNS since it doesn't really make sense for other systems. Use the `use_control_systems` bool to determine if you should add this trigger or not (that's kind of been the sentinel bool for BNS in these metavars)